css/css-lists/nested-list-with-list-style-type-none.html

```diff
@@ -19,6 +19,7 @@
     </li>
   </ul>
   <script type="text/javascript">
+    setup({ single_test: true });
```

Comment:
Can you add a TODO to rewrite this after https://github.com/web-platform-tests/rfcs/pull/33?

Reply:
Sure